packages/relay/src/lib/types/IMirrorNode.ts

```diff
@@ -18,6 +18,28 @@
  *
  */
 
+export interface IAccountInfo {
+  /**
+   * Account ID string in the form `shard.realm.num`
+   */
+  account: string;
+  /**
+   * RFC4648 no-padding base32 encoded string of the account's alias.
+   */
+  alias: string;
+  balance?: IAccountBalance;
+  deleted?: boolean;
+  ethereum_nonce?: number;
```

Comment:
Should we consider making the nonce non-optional with a default value of zero?  Could adding JSDoc comments help with documentation?

Reply:
It’s important to note that interfaces in TypeScript are just for defining the structure of the data, like a blueprint. They don’t actually exist after the code is compiled to JavaScript, so you can't set default values in them.

If we want a default value for the nonce, like zero, we’d need to create a class that implements this interface. Then, we would be able to set the default value in the class's constructor. But I think it’s better to keep the original data that is coming from the mirror node, without modifying it unless we have a good reason to.

As for adding JSDoc comments, that’s a great idea! They’ll definitely help with documentation and make the code easier to understand.